config/src/config/mod.rs

```diff
@@ -275,17 +275,14 @@ impl NodeConfig {
         self.mempool.randomize_ports();
         self.storage.randomize_ports();
     }
-}
 
-pub struct NodeConfigHelpers {}
+    pub fn random() -> Self {
```

Comment:
Since this is only used for tests, why not `for_test()` ? I don't like it being called `random` since it is not random : o

Reply:
this is a good point.... we already have clone_for_template, so we should make this also explicit.